src/features/query-bar/flowTypes.js

```diff
@@ -1,11 +1,13 @@
 // @flow
-import { AND, OR } from './constants';
+import { AND, OR, EMPTY_CONNECTOR } from './constants';
 
-export type SelectOptionType = {
+export type ConnectorType = typeof AND | typeof OR | typeof EMPTY_CONNECTOR;
+
+export type OptionType = {
     displayText: string,
     fieldId: string,
     type?: string,
-    value: string,
+    value: ConnectorType,
```

Comment:
Not sure if this type is correct. I tried using string | ConnectorType but that still resulted in an error for `handleConnectorChange` when trying to setState with `connector`.

Reply:
Using ConnectorType means that value can only be the string literals AND / OR and other strings are not allowed according to the flow message. Using Union of (string | ConnectorType) should technically allow value to be any string or AND or OR but flow still doesn't seem to like it.

Reply:
Did you try just `string`? when you do `string | ConnectorType` that basically boils down to `string`